src/plugin/parse-sitemap/src/java/org/apache/nutch/parse/sitemap/NutchSitemapParser.java

```diff
@@ -62,7 +68,6 @@ public NutchSitemapParse getParse(String url, WebPage page) {
       LOG.error(StringUtils.stringifyException(e));
     }
     Map<Outlink, Metadata> outlinkMap = null;
-    Iterator i$;
     if (siteMap.isIndex()) {
```

Comment:
Can someone walk me through how sitemap indexes are supposed to be being handled?
It doesn't seem to me like sitemaps in the WebPage.sitemaps field are ever actually added to the crawler store. And if they are, they are not marked with the sitemap inject marker.

Reply:
Here too: https://github.com/apache/nutch/pull/304/files#diff-44035662a89a7d5fb73289b33bc0dab9R539
I don't see where this is actually adding the sitemap to the datastore for generate jobs.

Reply:
The only other place the WebPage.sitemaps field is used is here: https://github.com/benmvachon/nutch/blob/f314baf3d346f58c407977b65283fcf3e23055b5/src/java/org/apache/nutch/crawl/DbUpdateMapper.java#L72